src/main/java/com/fasterxml/jackson/databind/util/StdDateFormat.java

```diff
@@ -285,38 +274,20 @@ public boolean isLenient() {
      */
 
     @Override
-    public Date parse(String dateStr) throws ParseException
+    public Date parse(final String source) throws ParseException
     {
-        dateStr = dateStr.trim();
-        ParsePosition pos = new ParsePosition(0);
-
-        Date dt;
+        String dateStr = source.trim();
 
         if (looksLikeISO8601(dateStr)) { // also includes "plain"
-            dt = parseAsISO8601(dateStr, pos, true);
-        } else {
-            // Also consider "stringified" simple time stamp
```

Comment:
Is this removed because it is handled at higher level, by deserializer (and not `DateFormat` implementation)?